Sources/SwiftProtobuf/NameMap.swift

```diff
@@ -8,147 +8,217 @@
 //
 // -----------------------------------------------------------------------------
 
-// Proto names are always ASCII and are always literals, so we can improve the
-// performance of some of our parsing by storing them as pointers to the
-// underlying bytes of a StaticString to avoid the overhead of creating and
-// comparing Unicode Strings.
+import Darwin
```

Comment:
No longer needed:  I was using malloc/free to manage the memory for caching UTF8 bytes, but I changed it to use UnsafeMutableRawPointer instead.